examples/database/libsql/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3"
+services:
+  sqld-primary:
+    image: ghcr.io/libsql/sqld:main
+    environment:
+      - SQLD_NODE=primary
+      - SQLD_HTTP_LISTEN_ADDR=0.0.0.0:8000
+    volumes:
+      - "/tmp/data.db:/var/lib/sqld/iku.db"
+  sqld-replica:
+    image: ghcr.io/libsql/sqld:main
+    depends_on:
+      - sqld-primary
+    environment:
+      - SQLD_NODE=replica
+      - SQLD_PRIMARY_URL=http://sqld-primary:5001
+    volumes:
+      - "/tmp/replica.db:/var/lib/sqld/iku.db"
+  flipt-one:
+    image: flipt:dev-cb18b18
```

Comment:
im guessing this is from your Dagger build? maybe we can use our relative Dockerfile for now? at `../../../Dockerfile`? (project root)

Reply:
@markphelps Yeah meant to specify that point, apologies!